pyogrio/tests/test_raw_io.py

```diff
@@ -340,9 +340,15 @@ def assert_equal_result(result1, result2):
     # a plain `assert np.array_equal(geometry1, geometry2)` doesn't work because
     # the WKB values are not exactly equal, therefore parsing with pygeos to compare
     # with tolerance
-    pygeos = pytest.importorskip("pygeos")
-    assert pygeos.equals_exact(
-        pygeos.from_wkb(geometry1), pygeos.from_wkb(geometry2), tolerance=0.00001
+    try:
```

Comment:
Maybe consider flipping the precedence to try with shapely first and then fall back to pygeos (makes it easier to drop pygeos altogether once shapely2 has been out for a while)